qp_qiime2/__init__.py

```diff
@@ -76,234 +86,191 @@
 
         for m in methods:
             # after review of qiime2-2019.4 we decided to not add these methods
-            if (q2plugin.name, m.id) in [('feature-table', 'group'),
-                                         ('feature-table', 'filter_seqs')]:
-                continue
+            if (q2plugin.name, m.id) not in [('feature-table', 'group'),
+                                             ('feature-table', 'filter_seqs')]:
+                methods_to_add.append((q2plugin, m))
 
-            inputs = m.signature.inputs.copy()
-            outputs = m.signature.outputs.copy()
-            parameters = m.signature.parameters.copy()
-            add_method = True
+for q2plugin, m in methods_to_add:
```

Comment:
Once we have all the `methods_to_add `, we simply do what we used to do in the block above - however, the code and ident had to slightly change to  accommodate the new structure.